Source/WebCore/css/typedom/CSSUnitValue.cpp

```diff
@@ -186,6 +189,59 @@ RefPtr<CSSValue> CSSUnitValue::toCSSValue() const
     return CSSPrimitiveValue::create(m_value, m_unit);
 }
 
+static bool isValueOutOfRangeForProperty(CSSPropertyID propertyID, double value, CSSUnitType unit)
```

Comment:
This doesn't look exhaustive. For instance `perspective` is not in here. I guess this was informed by the set of properties used in existing WPT tests?

Reply:
No, this wasn't informed by WPT tests. This matches exactly what Chrome is doing as I couldn't find a spec for this.